### DISCUSSION TITLE
How to get the creator's address of a contract?

### DISCUSSION THREAD

**Discussion by johnson86tw** [wevm/viem#887] [Question] [answered]
I hope to find the address of the factory contract through a contract address. How can I identify the address from which a contract was deployed in viem?

The problem is something like this https://ethereum.stackexchange.com/questions/12903/find-the-creator-of-a-contract

**Comment by jxom** [comment, marked as answer]
You’ll need to use an indexer (most common one is [Etherscan](https://docs.etherscan.io/api-endpoints/contracts#get-contract-creator-and-creation-tx-hash) ).

**Reply by johnson86tw** [reply]
Understood. Thank you!
